go/vt/schemamanager/tablet_executor.go

```diff
@@ -357,8 +363,16 @@ func (exec *TabletExecutor) Execute(ctx context.Context, sqls []string) *Execute
 	}
 	providedUUID := ""
 
+	rl := timer.NewRateLimiter(lockRenewTimeout)
+	defer rl.Stop()
```

Comment:
Nice to see this getting more usage! I will eventually use it for something too... just have to remember. 🙂